docs/source/definition-of-done.md

```diff
@@ -0,0 +1,12 @@
+# Definiton of done
```

Comment:
```suggestion
# Definition of done for internal Jira Issues
```

This definition of done is so old, internal and not used at all. I would rather remove it completely from the upstream project documentation and mention it for example in https://docs.google.com/document/d/1T_HQ1HoyjLlHcdNGlKp_D17Qs3s5strR0Y_y2mg8rBs/.

Do @GAZDOWN, @pirat89, @drehak and @ZhukovGreen agree?

Reply:
I agree. This doesn't make sense to me. Our interanl stuff should remains internal.

Reply:
ping  @GAZDOWN, @pirat89, @drehak :) I am about to remove it. Do you agree?